app/services/idv/steps/document_capture_step.rb

```diff
@@ -1,9 +1,31 @@
 module Idv
   module Steps
     class DocumentCaptureStep < DocAuthBaseStep
-      def call; end
+      def call
+        response = post_images
+        if response.success?
+          mark_selfie_step_complete_unless_liveness_checking_is_enabled
```

Comment:
I think we can pull this out for now. Eventually this step will also handle the selfie. For now it will break without the selfie because that depends on the instance ID.

Reply:
OK